virtool/account/db.py

```diff
@@ -63,6 +65,37 @@ async def get_alternate_id(db, name: str) -> str:
         suffix += 1
 
 
+async def fetch_complete_key(mongo, key_id: str) -> Optional[APIKey]:
```

Comment:
Needs brief docstring.

```suggestion
async def fetch_complete_api_key(mongo, key_id: str) -> Optional[APIKey]:
```